builtin/types/user_deployer_whitelist/user_deployer_whitelist.proto

```diff
@@ -50,10 +50,12 @@ message UserDeployerState {
     Address address = 1;
     repeated DeployerContract contracts = 2;
     TierID tier_id = 3 [(gogoproto.customname)="TierID"];
+    bool active = 4;
```

Comment:
`active` -> `inactive`, default value is `false`, and this protobuf is already used in production so all existing data will have this set to `false`.

Reply:
done